src/rans/rANSCoder.py

```diff
@@ -50,10 +56,15 @@ def find_in_int_dist(cdf, to_find):
 
     print ("Error: Could not find symbol in integer-dist")
 
+spec = [
+    ('state', numba.int64),
+    ('encoded_data', types.ListType(types.int64)),
+]
+@numba.experimental.jitclass(spec=spec)
```

Comment:
I think that may be the docs for an old version of numba - the latest version as far I can tell is 0.56.1, where it looks to be regarded as an experimental feature https://numba.readthedocs.io/en/stable/user/jitclass.html

Reply:
ah okay, fair enough :+1: - if the module works without breaking I'm fine with using an experimental feature, I'll just specify the requirements to use a specific version of numa for pip, so that the code won't break with an updated version of numa.